### DISCUSSION TITLE
训练题目删除Bug

### DISCUSSION THREAD

**Discussion by toprise** [hydro-dev/Hydro#867] [General]
创建一个训练后，不小心把这个训练中引用了的题目删除了，导致训练无法正常，而且也没法删除，建议优化一下，比如说在训练中的题目标记一下已删除，不要直接打不开，或者，在删除题目时检查一下是否在某个训练里面，如果在，提醒一下也好，不然得重新创建训练。

**Comment by Molmin** [comment]
> 创建一个训练后，不小心把这个训练中引用了的题目删除了，导致训练无法正常，而且也没法删除，建议优化一下，比如说在训练中的题目标记一下已删除，不要直接打不开，或者，在删除题目时检查一下是否在某个训练里面，如果在，提醒一下也好，不然得重新创建训练。

链接后面加 `/edit` 可以修改题目

**Comment by toprise** [comment]
哦，谢谢。
